packages/expo-passkeys/package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "@clerk/expo-passkeys",
+  "version": "0.0.1",
+  "description": "Passkeys library to be used with Clerk for expo",
+  "keywords": [
+    "react-native",
+    "expo",
+    "expo-passkey",
+    "ClerkExpoPasskeys",
+    "clerk",
+    "passkeys"
+  ],
+  "homepage": "https://clerk.com/",
+  "bugs": {
+    "url": "https://github.com/clerk/javascript/issues"
+  },
+  "repository": "git+https://github.com/clerk/javascript.git",
+  "license": "MIT",
+  "author": "Clerk",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "scripts": {
+    "build": "EXPO_NONINTERACTIVE=1 expo-module build",
+    "build:watch": "expo-module build",
+    "clean": "expo-module clean",
+    "expo-module": "expo-module",
+    "lint": "expo-module lint",
+    "open:android": "open -a \"Android Studio\" example/android",
+    "open:ios": "xed example/ios",
+    "prepare": "expo-module prepare",
+    "prepublishOnly": "expo-module prepublishOnly"
+  },
+  "dependencies": {
+    "@clerk/shared": "2.10.1-snapshot.va96a299",
+    "@clerk/types": "4.28.0-snapshot.va96a299"
+  },
+  "devDependencies": {
+    "expo-module-scripts": "^3.5.2",
+    "expo-modules-core": "^1.12.19"
+  },
+  "peerDependencies": {
+    "expo": "*",
+    "react": "*",
+    "react-native": "*"
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
+}
```

Should we drop this?

---

👆

---

Dropping this or replacing with npm is causing the `expo-module-script` to fail.

---

Why though? what's the error?

---

```
npm error code 1
npm error path /Users/alexandrosntitoras/Desktop/clerk/javascript/packages/expo-passkeys
npm error command failed
npm error command sh -c expo-module prepare
npm error Configuring module
npm error expo-module-scripts: README.md exists, not updating
npm error /Users/alexandrosntitoras/.nvm/versions/node/v20.15.1/lib/node_modules/corepack/dist/lib/corepack.cjs:23512
npm error               throw new UsageError(`This project is configured to use ${result.spec.name} because ${result.target} has a "packageManager" field`);
```
This is the error if I remove it.

---

@AlexNti  If we do the following at least will it work?
```suggestion
  "packageManager": "yarn@1.22.22"
}
```

---

This seems to work, but why to choose this over the one generated from expo?

---

Was this field autogenerated by Expo?

---

Yes, this generated by the expo-modules-script